app/frontend/users/UserEdit.res

```diff
@@ -12,6 +12,7 @@ type state = {
   about: string,
   locale: string,
   email: string,
+  newEmail: string,
```

Comment:
Refer to the comments below to have this updated to an option<string> field initialised based on the `new_email` column in the users table.